src/olympia/tags/views.py

```diff
@@ -0,0 +1,24 @@
+from django.db.transaction import non_atomic_requests
+from django.utils.cache import patch_cache_control
+
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .models import Tag
+
+
+class TagListView(APIView):
+    authentication_classes = []
+
+    def get(self, request, *args, **kwargs):
+        return Response(list(Tag.objects.values_list('tag_text', flat=True)))
```

Comment:
Should we return these sorted, or is that done automatically based on a model annotation?

Reply:
the model defines a default order (by `tag_text`, which is what we want)